src/Core/Animation/DualQuaternionSkinning.cpp

```diff
@@ -1,12 +1,14 @@
 #include <Core/Animation/DualQuaternionSkinning.hpp>
 
+#include <Core/Animation/SkinningData.hpp>
+
 namespace Ra {
 namespace Core {
 namespace Animation {
-void computeDQ( const Pose& pose, const Sparse& weight, DQList& DQ ) {
+
+DQList computeDQ( const Pose& pose, const Sparse& weight ) {
```

Comment:
What is the benefit of returning local object from the function ? Does this introduce any overhead ?

Reply:
no overhead due to RVO (or copy elision). see https://en.cppreference.com/w/cpp/language/return#Notes

Reply:
Hum ok, fine.